pkg/util/constant.go

```diff
@@ -37,6 +37,8 @@ const (
 	LocalHostpathCasType = "localpv-hostpath"
 	// LocalDeviceCasType cas type name
 	LocalDeviceCasType = "localpv-device"
+	// Local host path cas type name
+	LocalHostpath = "local-hostpath"
```

Comment:
`LocalHostpath = "local-hostpath"` and `LocalHostpathCasType = "localpv-hostpath"
are bit confusing in the first look and maybe error prone in future, Can We change the name, because `LocalHostpath` actually shows the cas type of Local-Hostpath

Reply:
question. why are we adding a new type called "LocalHostpath = "local-hostpath"?

Reply:
Because that is the cas-type for showing local-hostpath volumes.
When Jiva volumes are created, they create the underlying replicas and PV the PV that it creates is the `local-hostpath` with the storage class name as `openebs-hostpath`.  This volume is actual Persistent volume that has a claim of the jiva volumes. (acc to my understanding)
Earlier We could not describe the `localhostpath` cas types, this PR does the work to list and describe those PVs.

Reply:
Hmm.. so is the following accurate:
- localpv-hostpath ( will display the details for PVCs created by users)
- local-hostpath ( will be used for displaying backend PVCs?) 

Can we extend `localpv-hostpath` itself to cover both cases. Can we do this, if we set a annotation via the jiva storage classes that says, that PVC is a backend PVC for a given jiva PVC?

Let us see if we can avoid introducing additional cas type in this cas.

Reply:
What I think is:
- local-hostpath is the actual local volume that is provisioned by openebs
- It's case type is: `local-hostpath`
- It has path on the cluster's disk where the data resides.
- So it the the PV and other CSI have a claim over this volume

Currently having less knowledge about localPV, I think @vharsh could clarify further details with more clarity :smiley:

Reply:
IMO, another cas-type `local-hostpath` is not required and might as well be confusing. Jiva volume provisioner creates a replicated volume backed by `n` local volumes on different nodes which are provisioned by [dynamic-localpv-provisioner](https://github.com/openebs/dynamic-localpv-provisioner/).

If a user provisions a Local hostpath backed volume, they are provisioned by the same [dynamic-localpv-provisioner](https://github.com/openebs/dynamic-localpv-provisioner/) just that now it's happening directly(not via jiva-operator component).

Reply:
@kmova I need to analyze once why `localpv-hostpath` was present earlier. We wouldn't need both `local-hostpath` and `localpv-hostpath` as both of them are one-and-the-same-thing.



But as an enhancement, we can add some flag or key in the describe output of LocalPV-hostpath which can provide some clarity towards the case you've pointed out, so that the user can ascertain that the first one is something some non-OpenEBS application(like MySQL, MongoDB consuming localpv-hostpath volumes) is directly using and the second one is something created by OpenEBS provisioners to support some non-OpenEBS application(like MySQL, MongoDB consuming Jiva volumes). 

> 
    - localpv-hostpath ( will display the details for PVCs created by users)
    - local-hostpath ( will be used for displaying backend PVCs?)

Reply:
We can enhance the jiva operator code to put an annotation of something on the PVC/PV launched for jiva replicas. Can you please check if it doesn't exist already.

Reply:
@kmova Both the constants are required, because the LocalPV-hostpath PV has this label `openebs.io/cas-type=local-hostpath`